content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -748,6 +749,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `VolumeCapacityPriority`: Enable support for prioritizing nodes in different
   topologies based on available PV capacity.
 - `WatchBookmark`: Enable support for watch bookmark events.
+- `WatchList` : Enable support for streaming initial state of objects in watch requests.
```

Comment:
If you want, hyperlink to `/docs/reference/using-api/api-concepts/#streaming-lists` here.

Reply:
I don't see hyperlinks to really be used here, so leaving as is for now.

Reply:
As a maintainer for the docs SIG, I'm specifically requesting you consider this addition. We'd like to see more links as these help readers.

You don't have to, but please don't use “other entries don't have this“ as grounds to consider it as unacceptable.

Reply:
done